CHANGELOG.md

```diff
@@ -958,14 +977,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - CHANGELOG_CE.md: update date
 
 
-## [1.0.1] - 2022-09-02
```

Comment:
Why thi change?

Reply:
This version doesn't have link (it's first tag) so it's displayed like that:

![image](https://github.com/user-attachments/assets/06511ac5-64c5-4bfc-b2e8-708d82a42778)

compare to:

![image](https://github.com/user-attachments/assets/ba5feee2-940d-4465-9c17-38efdbb279c5)

Reply:
I guess I could link to <https://github.com/Dasharo/meta-dts/compare/035081699ca860e8a315e98893830b67a817730a..v1.0.1> instead